chain/actions/src/payload.rs

```diff
@@ -103,6 +104,16 @@ fn channels_payload(hopr_channels: Address, call_data: Vec<u8>) -> Vec<u8> {
     .abi_encode()
 }
 
+fn multisend_payload(call_data: Vec<u8>) -> Vec<u8> {
+    execTransactionFromModuleCall {
+        to: MULTISEND_CALL_ONLY,
+        value: U256::ZERO,
+        data: call_data.into(),
+        operation: Operation::DelegateCall as u8,
+    }
+    .abi_encode()
+}
+
```

Comment:
A more elegant approach would be to have `Payload` as a builder and then generate the `Vec<u8>` using a function explicitly, e.g. `as_single_call()` or `as_multi_call`.